examples/gogo/app/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>gogo</artifactId>
+    <groupId>org.osgi.enroute.examples.gogo</groupId>
+    <version>0.0.2-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>app</artifactId>
+
+  <description>The application packaging project for the Gogo OSGi enRoute example</description>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.osgi.enroute.examples.gogo</groupId>
+      <artifactId>command</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.osgi.enroute</groupId>
+      <artifactId>osgi-api</artifactId>
+      <type>pom</type>
+    </dependency>
+    <dependency>
+      <groupId>org.osgi.enroute</groupId>
+      <artifactId>impl-index</artifactId>
+      <type>pom</type>
+    </dependency>
+    <dependency>
+      <groupId>org.osgi.enroute</groupId>
+      <artifactId>debug-bundles</artifactId>
+      <type>pom</type>
+      <scope>runtime</scope>
```

Comment:
The debug-bundles and so Gogo are part of the test scope at the moment.
Should we add a direct dependency to the Gogo ones here or should we change the scope of the debug-bundles?

Reply:
I think that in this case it may make more sense to add the Gogo bundles separately, if only so that we can easily target newer versions containing the implementation capability later.